tests/storages_tests/rdb_tests/create_db.py

```diff
@@ -24,6 +65,20 @@ def mo_objective_test_upgrade(trial: optuna.trial.Trial) -> Tuple[float, float]:
     return x, x**2 + y**2 + z**2
 
 
+def objective_test_upgrade_v3_distribution(trial: optuna.trial.Trial) -> float:
```

Comment:
Thanks for the update of the name. I prefer to remove the `v3`.

> BTW, do you mean we will add a parametrization on test_upgrade_distributions after v3.0.0?

I'm not sure, but I think it is safe to test the later version of Optuna does not violate the instantiation rule of distributions. I think we can add the later version of Optuna in this parametrization.

Reply:
Fair enough. 9ff961b revised the name of an objective as you originally suggested.